lib/adapter.js

```diff
@@ -700,6 +700,7 @@ AbstractPouchDB.prototype.info = utils.adapterFun('info', function (callback) {
     // assume we know better than the adapter, unless it informs us
     info.db_name = info.db_name || self._db_name;
     info.auto_compaction = !!(self.auto_compaction && self.type() !== 'http');
+    info.adapter = info.adapter || self.type();
```

Comment:
`type()` already has logic to fallback so info.adapter, so I think we should stick to 

    info.adapter = self.type();

(and scold ourself for calling it `.type()` in the first place